custom/intrahealth/__init__.py

```diff
@@ -144,7 +144,7 @@ def _get_location(form):
         if not user_id:
             return None
         try:
-            user = CommCareUser.get(user_id)
+            user = CouchUser.wrap_correctly(CouchUser.get_db().get(user_id))
```

Comment:
nit: [`CouchUser.get_by_user_id`](https://github.com/dimagi/commcare-hq/blob/4bfb18d4f2ba2588ecbd5a905c76331f916fec43/corehq/apps/users/models.py#L1161-L1177) does this.